root/etc/s6-overlay/s6-rc.d/init-mod-netbox-slurpit-install/run

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/with-contenv bash
```

Comment:
this service file needs to depend on `init-mods` in order to run before package install script

Reply:
I am not sure I understand. I can see in the docker logs locally 

```
[mod-init] Running Docker Modification Logic
[mod-init] Adding linuxserver/mods:pull_request_944 to container
[mod-init] Downloading linuxserver/mods:pull_request_944 from lscr.io
[mod-init] Installing linuxserver/mods:pull_request_944
[mod-init] linuxserver/mods:pull_request_944 applied to container
```

Is that what you are referring to?

Reply:
disregard. I figured this out